msticpy/data/uploaders/splunk_uploader.py

```diff
@@ -17,19 +17,20 @@
 from .uploader_base import UploaderBase
 
 __version__ = VERSION
-__author__ = "Pete Bryan"
+__author__ = "Pete Bryan, Tatsuya Hasegawa"
 
 
 class SplunkUploader(UploaderBase):
     """Uploader class for Splunk."""
 
-    def __init__(self, username: str, host: str, password: str, **kwargs):
+    def __init__(self, **kwargs):
         """Initialize a Splunk Uploader instance."""
         super().__init__()
         self._kwargs = kwargs
-        self.workspace = host
-        self.workspace_secret = password
-        self.user = username
+        self.host = kwargs.get("host")
+        self.password = kwargs.get("password")
+        self.username = kwargs.get("username")
+        self.splunkToken = kwargs.get("splunkToken")
```

Comment:
Same here - rename param and attribute to use snake casing. 
param can be `token` but attribute could be splunk_token or token

Reply:
Checked and Done!